server/app/modules/DurableJobModule.java

```diff
@@ -87,7 +87,7 @@ public DurableJobRegistry provideDurableJobRegistry(
         DurableJobName.REPORTING_DASHBOARD_MONTHLY_REFRESH,
         persistedDurableJob ->
             new ReportingDashboardMonthlyRefreshJob(reportingRepository, persistedDurableJob),
-        new RecurringJobExecutionTimeResolvers.FirstOfMonth2Am());
+        new RecurringJobExecutionTimeResolvers.Sunday2Am());
```

Comment:
(blocking) If we're changing the refresh rate the job name needs to be renamed otherwise it's confusing when you see it happening weekly. Change it from `REPORTING_DASHBOARD_MONTHLY_REFRESH` to `REPORTING_DASHBOARD_REFRESH` then it won't matter what we make the refresh rate. Change the enum value and the class name please.